File: java/785A.java
// 785 A. Anton and Polyhedrons

import java.util.*;

class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        Map<String, Integer> faces = new HashMap<>();
        faces.put("Tetrahedron", 4);
        faces.put("Cube", 6);
        faces.put("Octahedron", 8);
        faces.put("Dodecahedron", 12);
        faces.put("Icosahedron", 20);

        long total = 0;
        for (int i = 0; i < n; i++) {
            String poly = sc.next();
            total += faces.get(poly);
        }

        System.out.println(total);
    }
}

File: cpp/785A.cpp
// 785 A. Anton and Polyhedrons

#include <bits/stdc++.h>
using namespace std;

int main() {
    int n;
    cin >> n;

    unordered_map<string, int> faces = {
        {"Tetrahedron", 4},
        {"Cube", 6},
        {"Octahedron", 8},
        {"Dodecahedron", 12},
        {"Icosahedron", 20}
    };

    long long total = 0;
    string poly;
    for (int i = 0; i < n; i++) {
        cin >> poly;
        total += faces[poly];
    }
    
    cout << total;
    return 0;
}
